config.dev.toml

```diff
@@ -341,15 +354,9 @@ SingleFlightType = "memory"
         # Env override: ATHENS_S3_BUCKET_NAME
         Bucket = "MY_S3_BUCKET_NAME"
 
-        # If true then the default aws configuration will be used. This will
-        # attempt to find credentials in the environment, in the shared
-        # configuration (~/.aws/credentials) and from ec2 instance role
-        # credentials. See
-        # https://godoc.org/github.com/aws/aws-sdk-go#hdr-Configuring_Credentials
-        # and
-        # https://godoc.org/github.com/aws/aws-sdk-go/aws/session#hdr-Environment_Variables
-        # for environment variables that will affect the aws configuration.
-        UseDefaultConfiguration = false
+        # https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/endpointcreds/
+        AWS_CREDENTIALS_ENDPOINT = ""
```

Comment:
This key name here is named differently from the way it's defined in this struct, so it will never got unmarshaled correctly as far as I can see. Right?